src/Evolution/Systems/Ccz4/Christoffel.hpp

```diff
@@ -70,4 +70,33 @@ tnsr::Ijj<DataType, Dim, Frame> christoffel_second_kind(
     const tnsr::i<DataType, Dim, Frame>& field_p,
     const tnsr::Ijj<DataType, Dim, Frame>& conformal_christoffel_second_kind);
 /// @}
+
+/// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the contraction of the conformal spatial christoffel symbols
```

Comment:
`christoffel` -> `Christoffel` (they're named after someone :) ), also in the tag